pkg/agent/ops.go

```diff
@@ -26,34 +27,37 @@ import (
 	"github.com/litmuschaos/litmusctl/pkg/utils"
 )
 
+var (
+	cyan = color.New(color.FgCyan, color.Bold)
+	red  = color.New(color.FgRed)
```

Comment:
since these are used in all the files for printing and formatting we can keep them in a separate package called `terminal` or `console` where we can keep common formatting styles, the unicodes for emojis, other utilities like printing a table etc in the future as well.

Reply:
Done